.github/workflows/integration.yml

```diff
@@ -44,3 +44,50 @@ jobs:
       with:
         nix_path: nixpkgs=channel:nixos-20.09
     - run: nix-shell third-party/nix/shell.nix --run "cargo build --release"
+
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hecrj/setup-rust-action@v1
+      with:
+        rust-version: 'nightly-2021-06-29'
+        targets: 'wasm32-unknown-unknown'
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+
+    - name: Docker meta
+      id: docker_meta
+      uses: crazy-max/ghaction-docker-meta@v1
+      with:
+        images: staketechnologies/astar-collator
```

Comment:
The Docker Hub repo seems to be the same with `development/shibuya` branch https://github.com/PlasmNetwork/Astar/blob/development/shibuya/.github/workflows/integration.yml#L70

Can I remove `development/shibuya` and rename `production/shiden` to `development/shiden`?

Reply:
I’m agree